sky/packages/sky/lib/base/scheduler.dart

```diff
@@ -34,14 +35,19 @@ void beginFrame(double timeStamp) {
     callback(timeStamp);
 }
 
+/// Registers [beginFrame] callback with the engine.
 void init() {
   sky.view.setFrameCallback(beginFrame);
 }
 
+/// Call callback every every frame.
```

Comment:
I think you were getting tired when you got to this file. :-P

first every should be for? after? before?